doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -117,7 +84,7 @@ Overview
 
    # hack hack
    git add my_new_file
-   git commit -am 'ENH: some message'
+   git commit -a
```

Comment:
It is also good practice to do a `git status` before making commits to see what is going to happen, sometimes a `git diff` also. Both can help avoid mistakes.